### PR TITLE
App openupify

### PR DESCRIPTION
**Pull request by guushoekman**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Code for South Africa
+Copyright (c) 2017 OpenUp
```
Comment (longhotsummer) [inline LICENSE]: Copyright (c) 2016, 2017 OpenUp

File: README.md [hunk 2]
```diff
@@ -48,14 +48,16 @@ Phonegap is built on [Phonegap Build](https://build.phonegap.com/apps/2227365/bu
 
 # License and Copyright
 
-Copyright 2016 Code for South Africa.
+Copyright 2017 OpenUp.
```
Comment (longhotsummer) [inline README.md]: 2016, 2017

File: README.md [hunk 3]
```diff
@@ -48,14 +48,16 @@ Phonegap is built on [Phonegap Build](https://build.phonegap.com/apps/2227365/bu
...
 
 Licensed under the [MIT license](LICENSE).
 
-The Pocket Report name and logo are Copyright 2016 Code for South Africa, and may not be used without permission.
+The Pocket Report name and logo are Copyright 2017 OpenUp, and may not be used without permission.
```
Comment (longhotsummer) [inline README.md]: Pocket Report_er_ ... 2016

**Review by longhotsummer — CHANGES_REQUESTED** [review]
Minor tweaks. When stating copyright, state the initial year and any years with substantial changes.

File: LICENSE (hunk 1 above)
Comment (guushoekman) [reply LICENSE]: Ah cool, wasn't sure what to do with that

**Comment by guushoekman** [comment]
Tweaks made. Don't want to merge quite yet, don't know if this immediately impacts the app or something.

**Review by longhotsummer — APPROVED** [review]

**Comment by longhotsummer** [comment]
Looks good. It's safe to merge, the app build and release process is manual.

**Pull request merged**